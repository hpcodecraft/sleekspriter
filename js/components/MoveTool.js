var MoveTool = React.createClass({
  render: function() {
    return (
      <div id="Move-Tool" className="ToolComponent">
        <i className="icon flaticon-move11"></i>

        <span className="hint">Move pixels of a layer by dragging.</span>
      </div>
    );
  }
});