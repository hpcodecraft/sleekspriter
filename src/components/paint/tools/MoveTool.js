import React from "react";
import { t } from "../../../utils";

const MoveTool = () => {
  return (
    <div id="Move-Tool" className="ToolComponent">
      <i className="icon flaticon-move11" />
      <span className="hint">{t("Move pixels of a layer by dragging.")}</span>
    </div>
  );
};

export default MoveTool;
