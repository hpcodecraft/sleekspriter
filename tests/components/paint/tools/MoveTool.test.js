import React from "react";
import MoveTool from "components/paint/tools/MoveTool";

describe("MoveTool", () => {
  it("should render correctly", () => {
    const component = shallow(<MoveTool />);
    expect(component.html()).toMatchSnapshot();
  });
});
