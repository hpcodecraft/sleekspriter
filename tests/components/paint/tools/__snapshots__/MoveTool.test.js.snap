// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoveTool should render correctly 1`] = `"<div id=\\"Move-Tool\\" class=\\"ToolComponent\\"><i class=\\"icon flaticon-move11\\"></i><span class=\\"hint\\">Move pixels of a layer by dragging.</span></div>"`;
